src/index.ts

```diff
@@ -0,0 +1,78 @@
+const inputElement = document.getElementById("commentInput");
+const formElement = document.getElementById("inputForm");
+const ulElement = document.getElementById("commentsList");
+
+//submit form
+if (formElement !== null) {
+    formElement?.addEventListener("submit", (event) => {
+        event.preventDefault();
+        if (inputElement !== null) {
+            const newMsg = (inputElement as HTMLInputElement).value;
+            if (newMsg === "") {
+                return;
+            }
+            // li father
+            const newLiElement = document.createElement("li");
+            // profile pic
+            const newImgElement = document.createElement("img");
+            newImgElement.className = "profilePicture";
+            newImgElement.src = "https://cdn.pixabay.com/photo/2021/05/20/11/58/beauty-6268460_1280.jpg";
+            newImgElement.alt = "profile Picture";
+            // comment 
+            const newCommentElement = document.createElement("div");
+            newCommentElement.className = "comonComment";
+            //comment Header
+            const newCommentHeaderElement = document.createElement("div");
+            const newCommentHeaderAndContentElement = document.createElement("div");
+            const newCommentBodyElement = document.createElement("span");
+            newCommentHeaderElement.className = "commentHeader";
+            newCommentBodyElement.innerText = ` ${newMsg}`;
+            const newCommentHeaderBoldElement = document.createElement("b");
+            newCommentHeaderBoldElement.innerText = "tonytoscani";
+            newCommentHeaderAndContentElement.append(newCommentHeaderBoldElement, newCommentBodyElement);
+            newCommentHeaderElement.append(newCommentHeaderAndContentElement);
+            // extra info
+            const newCommentInfoElement = document.createElement("div");
+            newCommentInfoElement.className = "commentInfo";
+            const newhourAgoElement = document.createElement("div");
+            newhourAgoElement.className = "infoSubtitle";
+            newhourAgoElement.innerText = "10 h";
+            const newlikesInfoElement = document.createElement("b");
+            newlikesInfoElement.className = "infoSubtitle";
+            newlikesInfoElement.innerText = "like";
+            const newReplayInfoElement = document.createElement("b")
+            newReplayInfoElement.className = "infoSubtitle";
+            newReplayInfoElement.innerText = "Reply";
+            newCommentInfoElement.append(newhourAgoElement, newlikesInfoElement, newReplayInfoElement);
+            newCommentHeaderElement.append(newCommentInfoElement)
+            // heart in the end
+            const newCommentSvgElement = document.createElement("img");
```

Comment:
Naming all theses variables "new" is a bit redundant, it should be made clear from context that this element was recently created